cmake/projects/libsodium/hunter.cmake

```diff
@@ -17,10 +18,14 @@ hunter_add_version(
     00a999629f1f8f500cef29818135f8f889f44e41
 )
 
+hunter_configuration_types(libsodium CONFIGURATION_TYPES Release)
 if(MSVC)
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_msbuild)
```

Comment:
This scheme works fine with build types: https://github.com/ruslo/hunter/blob/master/cmake/projects/libsodium/schemes/url_sha1_libsodium_msbuild.cmake.in